### PR TITLE
New releases

### PR DESCRIPTION
**Pull request by krasserm**
- akka-persistence-kafka 0.4
- akka-analytics 0.3
- streamz 0.3
- eventuate 0.2

### CODE REVIEW

File: community/index.md [hunk 1]
```diff
@@ -690,6 +690,27 @@ Before adding a plugin to this list, it should have been tested for compatibilit
...
+  - replicates application state by replicating events across multiple locations
+  - allows updates to replicated state at multiple locations concurrently (multi-master)
+  - allows individual locations to continue writing even if they are partitioned from other locations
+  - provides means to detect, track and resolve conflicting updates (automated and interactive)
+  - enables applications to implement a causal consistency model
+  - preserves causal ordering of replicated events
+  - provides implementations of operation-based CRDTs
+  - supports distribution up to global scale.
+
+  A comparison to Akka Pertsistence in given in [this article](https://krasserm.github.io/2015/05/25/akka-persistence-eventuate-comparison/). Find out more in the [project documentation](http://rbmhtechnology.github.io/eventuate/). Eventuate binaries are [published](http://rbmhtechnology.github.io/eventuate/download.html) to bintray at the URL `https://dl.bintray.com/rbmhtechnology/maven`.
```
Comment (ktoso) [inline community/index.md]: Typo in persistence :-)

**Comment by ktoso** [comment]
LGTM, please fix the typo :-)

**Comment by krasserm** [comment]
Fixed :-)

**Comment by patriknw** [comment]
thanks for the update, @krasserm

**Comment by krasserm** [comment]
@patriknw you're welcome :-)

**Pull request merged**